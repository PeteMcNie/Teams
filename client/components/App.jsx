import React from 'react'
import { Route } from 'react-router-dom'

import Title from './title'

const App = () => {
  return (
    <>
      <Route path='/' component={Title} />
    </>
  )
}

export default App
