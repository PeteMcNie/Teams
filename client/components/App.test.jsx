import React from 'react'

import { renderWithRedux } from '../testing/utils'

import App from './App'

test('App component displays correct components', () => {
  const { container } = renderWithRedux(<App />)
  expect(container).toMatchSnapshot()
})
