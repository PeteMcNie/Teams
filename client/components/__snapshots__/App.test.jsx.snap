// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component displays correct components 1`] = `
<div>
  <div>
    <h1
      class="title"
    >
      Teams
    </h1>
  </div>
  <div
    class="links"
  >
    <div
      class="link"
    >
      <a
        href="/createteams"
      >
        Create Teams
      </a>
    </div>
    <div
      class="link"
    >
      <a
        href="/addplayers"
      >
        Add New People
      </a>
    </div>
  </div>
  <div
    class="footer"
  >
    <div
      class="footericons"
    >
      <a
        href="https://github.com/PeteMcNie"
      >
        <i
          class="fab fa-github"
          id="github"
        />
      </a>
    </div>
    <div>
      P. R. McNie
    </div>
  </div>
</div>
`;
